src/exportToSheets.ts

```diff
@@ -95,4 +89,13 @@ export async function exportToSheets() {
       ],
     },
   });
+
+  // then clear the excess data
+  const start = values.length + 1;
+  const end = start + 10000;
```

Comment:
I would be tempted to pull the `10000` into a `const`, to explain what this magic value is doing.

Reply:
ok